azure-pipelines/test.yml

```diff
@@ -76,81 +76,3 @@ jobs:
       testResultsFiles: 'test/unit-test-results.xml'
       testRunTitle: '$(Agent.JobName)'
     condition: succeededOrFailed()
-
-- job: E2ETests
```

Comment:
Where is this being replaced by the new tests?

Reply:
https://github.com/Azure/azure-functions-nodejs-e2e-tests/blob/main/azure-pipelines/test.yml

The caveat is that it's not run as often, which might be what you're asking about. I'm not running it for PRs - just on a daily schedule. I think e2e tests are excessive for PRs